### PR TITLE
Use placeholder as default if field left empty and is required

### PR DESCRIPTION
**Pull request by hxnyk**
### Description:
If nothing's been entered for a required field, populate the value with the placeholder value & bug fixes @mcastorina 

### Checklist:
* [ ] Tests passing (`make test-community`)?
* [ ] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

File: pkg/tui/pages/source_configure/trufflehog_configure.go [hunk 1]
```diff
@@ -49,7 +50,7 @@ func GetTrufflehogConfiguration() truffleCmdModel {
 		Key:         "concurrency",
 		Required:    false,
 		Help:        "Number of concurrent workers.",
-		Placeholder: "1",
+		Placeholder: strconv.Itoa(runtime.NumCPU()),
```
Comment (rosecodym) [inline pkg/tui/pages/source_configure/trufflehog_configure.go:53]: which was the default concurrency before we added the tui stuff?

File: pkg/tui/pages/source_select/source_select.go [hunk 2]
```diff
@@ -73,6 +73,7 @@ func New(c common.Common) *SourceSelect {
 		OssItem("GCS (Google Cloud Storage)", "Scan a Google Cloud Storage instance."),
 		// Enterprise sources.
 		EnterpriseItem("Artifactory", "Scan JFrog Artifactory packages."),
+		// EnterpriseItem("Azure", "Scan Microsoft Azure resources."),
```
Comment (rosecodym) [inline pkg/tui/pages/source_select/source_select.go:76]: is this still supposed to be commented out?

File: pkg/tui/sources/github/github.go [hunk 3]
```diff
@@ -19,41 +19,69 @@ func GetFields() githubCmdModel {
...
-		command = append(command, "--repo="+inputs["repo"])
+	if inputs["repo"].Value != "" {
+		command = append(command, "--repo="+inputs["repo"].Value)
 	}
 
 	return strings.Join(command, " ")
 }
 
-func (m githubCmdModel) Summary() string {
+func (m githubCmdModel) Test() map[string]textinputs.Input {
```
Comment (rosecodym) [inline pkg/tui/sources/github/github.go:69]: is this function still supposed to be here?

File: pkg/tui/sources/syslog/syslog.go [hunk 4]
```diff
@@ -64,10 +64,8 @@ func (m syslogCmdModel) Cmd() string {
 	syslogKeys := [5]string{"address", "protocol", "cert", "key", "format"}
 
 	for _, key := range syslogKeys {
-		if inputs[key] != "" {
```
Comment (rosecodym) [inline pkg/tui/sources/syslog/syslog.go:67]: what makes this guard no longer necessary?

File: pkg/tui/pages/source_configure/trufflehog_configure.go (hunk 1 above)
Comment (mcastorina) [reply pkg/tui/pages/source_configure/trufflehog_configure.go:53]: The updated version. We copied it from here: https://github.com/trufflesecurity/trufflehog/blob/ed062178622c74167551eb8f2322c56a78d53a1d/main.go#L47

File: pkg/tui/sources/syslog/syslog.go (hunk 4 above)
Comment (hxnyk) [reply pkg/tui/sources/syslog/syslog.go:67]: Moved the check to the `GetInputs` func in `textinputs.go`

**Review by rosecodym — APPROVED** [review]
sorry, didn't notice that you were waiting on me

**Pull request merged**